app/assets/javascripts/sort.js

```diff
@@ -0,0 +1,4 @@
+ // Sortable.create(incomplete_tasks, {
```

Comment:
var incomplete_tasks = document. querySelectorAll(#allElementsForIncompleteTasks#) ;

update and put this above